proxy/proxy/rhnAuthCacheClient.py

```diff
@@ -186,9 +191,9 @@ def __str__(self):
     s = Shelf(('localhost', 9999))
     s['1234'] = [1, 2, 3, 4, None, None]
     s['blah'] = 'testing 1 2 3'
-    print 'Cached object s["1234"] = %s' % str(s['1234'])
-    print 'Cached object s["blah"] = %s' % str(s['blah'])
-    print s.has_key("asdfrasdf")
+    print('Cached object s["1234"] = {}'.format(str(s['1234'])))
```

Comment:
No need to `str()` this thing then. In Python3 bytes will be rendered as bytes repr, and string is just unicode string.